cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -22,13 +22,17 @@ import (
 
 const (
 	kubeDNSv170AndAboveVersion = "1.14.5"
```

Comment:
for clarity, you could update this to `kubeDNSv180Version`; kubeadm at HEAD will support v1.8 as the minimum supported version now